data/docs/messaging-queues/kafka.mdx

```diff
@@ -415,9 +415,417 @@ Run the collector with the above configuration :
+```
+
+### Example: Deploying Sample Kafka Producer and Consumer Applications with OpenTelemetry Java Agent
+
+```yaml
+# Producer Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-producer1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kafka-producer1
+  template:
+    metadata:
+      labels:
+        app: kafka-producer1
+    spec:
+      containers:
+      - name: producer
+        image: jaikanthjay46/kafka-consumer-test:latest
+        env:
+        - name: BOOTSTRAP_SERVERS
+          value: "signoz-demo-cluster1-kafka-bootstrap:9092"
+        - name: TOPIC
+          value: "topic1"
+        - name: PARTITION_KEY
+          value: "key1"
+        - name: DELAY
+          value: "10"
+        - name: OTEL_SERVICE_NAME
+          value: "producer-svc1"
+        - name: OTEL_TRACES_EXPORTER
+          value: "otlp"
+        - name: OTEL_METRICS_EXPORTER
+          value: "otlp"
+        - name: OTEL_LOGS_EXPORTER
+          value: "otlp"
+        - name: OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
+          value: "http://my-release-k8s-infra-otel-agent:4318/v1/logs"
+        - name: OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
+          value: "http://my-release-k8s-infra-otel-agent:4318/v1/metrics"
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: "http://my-release-k8s-infra-otel-agent:4318/v1/traces"
+---
+# Consumer Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-consumer1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kafka-consumer1
+  template:
+    metadata:
+      labels:
+        app: kafka-consumer1
+    spec:
+      containers:
+      - name: consumer
+        image: jaikanthjay46/kafka-consumer-test:latest
+        env:
+        - name: BOOTSTRAP_SERVERS
+          value: "signoz-demo-cluster1-kafka-bootstrap:9092"
+        - name: CONSUMER_GROUP
+          value: "cg1"
+        - name: TOPIC
+          value: "topic1"
+        - name: OTEL_SERVICE_NAME
+          value: "consumer-svc"
+        - name: OTEL_TRACES_EXPORTER
+          value: "none"
+        - name: OTEL_METRICS_EXPORTER
+          value: "none"
+        - name: OTEL_LOGS_EXPORTER
+          value: "none"
+        - name: OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
+          value: "http://my-release-k8s-infra-otel-agent:4318/v1/logs"
+        - name: OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
+          value: "http://my-release-k8s-infra-otel-agent:4318/v1/metrics"
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: "http://my-release-k8s-infra-otel-agent:4318/v1/traces"
+```
+
+Apply the deployments:
+```bash
+kubectl apply -f kafka-applications.yaml
+```
+Please take note of otel service name `my-release-k8s-infra-otel-agent` in the above yaml file, you will need to replace it with your otel service name.
+It's of the format `<helm-release-name>-k8s-infra-otel-agent`.
+
+Continue to visualize your data in SigNoz as described in the previous sections.
+
```

Comment:
Can you add an image of the final output in the messaging queue section that you see for Strimzi in this tab itself ?

Reply:
The article already has them, will add them when we separate it into different docs.

Reply:
Okay.